### PR TITLE
Add renew certificate github action

### PR DESCRIPTION
**Pull request by smiclea**
Runs the renew-certificate Juju action on the certbot-k8s charm every 2
months and on command.

### CODE REVIEW

**Comment by claudiubelu** [comment]
Depends On: https://github.com/pedroleaoc/certbot-k8s/pull/7

File: .github/workflows/renew_certificate.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Renew the CA-verified Certificate
```
Comment (claudiubelu) [inline .github/workflows/renew_certificate.yaml:1]: The Waltz Public Instance is on a regular EC2 instance, not on EKS (like the Legend Public instance). Which means that this action will not work in this case, we don't have the same means of connecting to it. Instead, the connection should be made similarly to how you've implemented the ``refresh_deployment`` github action.

Comment (smiclea) [reply .github/workflows/renew_certificate.yaml:1]: Updated accordingly.

**Comment by claudiubelu** [comment]
Looks good, we can merge once the necessary secrets have been added.

**Comment by claudiubelu** [comment]
The integration tests seem to fail because of microk8s, and is unrelated to this PR.

**Pull request merged**